src/System.Data.SqlClient/src/Interop/LocaleMapperApiHelper.cs

```diff
@@ -0,0 +1,482 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
We have different case as we have different implementation as I mentioned above. Windows in the future will use this mapping as well.

With your references, this case only  has different is the constant, UNIX, OSX, FreeBSD. 
https://github.com/dotnet/corefx/tree/master/src/System.Runtime.InteropServices.RuntimeInformation/src

But our case has different implementation, we lookup LCID with Windows API or using Hardcoded lookup. and as Windows is moving away from this LCID, in the future, both of them will use the same hardcoded mapping. or it may have different implementation.

This case will be similar with our SNINativeWrapper / Manage Helper.